decidim-proposals/app/commands/decidim/proposals/admin/create_proposal.rb

```diff
@@ -40,17 +40,23 @@ def call
         attr_reader :form, :proposal, :attachment
 
         def create_proposal
+          # parsed_title = Decidim::ContentProcessor.parse(form.title, current_organization: current_organization).rewrite
+          # parsed_body = Decidim::ContentProcessor.parse(form.body, current_organization: current_organization).rewrite
+
+          parsed_title = Decidim::ContentProcessor.parse_with_processor(:hashtag, form.title, current_organization: current_organization).rewrite
```

Comment:
Move these 2 lines inside the `attributes` method and you won't need to pass those as params